EIPS/eip-2794.md

```diff
@@ -94,8 +94,6 @@ As presented in the previous sections we can use the RICP algorithm to calculate
 ## Backwards Compatibility
 [EIP-2569](https://github.com/DAism2019/EIPs/blob/master/EIPS/eip-2569.md).
```

Comment:
Links should be relative:
```suggestion
 [EIP-2569](./eip-2569.md)
```
Also, you should explain how this EIP is related to 2569 in the EIP.